### PR TITLE
fix(lb): fix assignation of ip zone and make region computed

### PR DESCRIPTION
**Pull request by Monitob**

### CODE REVIEW

File: scaleway/helpers.go [hunk 1]
```diff
@@ -310,8 +315,8 @@ func regionComputedSchema() *schema.Schema {
 	}
 }
 
-// validateStringInSliceWithWarning helps to only returns warnings in case we got a non public locality passed
-func validateStringInSliceWithWarning(correctValues []string, field string) func(i interface{}, path cty.Path) diag.Diagnostics {
+// ValidateStringInSliceWithWarning helps to only returns warnings in case we got a non public locality passed
```
Comment (remyleone) [inline scaleway/helpers.go:318]: Why is this change required?

Comment (Monitob) [reply scaleway/helpers.go:318]: Not necessary anymore. I'll rollback this to private

File: scaleway/resource_lb_ip.go [hunk 2]
```diff
@@ -33,11 +33,17 @@ func resourceScalewayLbIP() *schema.Resource {
...
 			},
-			"region":          regionComputedSchema(),
-			"zone":            zoneSchema(),
+			"zone": {
+				Type:             schema.TypeString,
+				Description:      "The zone you want to attach the resource to",
+				Optional:         true,
+				ForceNew:         true,
+				ValidateDiagFunc: validateStringInSliceWithWarning(AllZones(), "zone"),
+				Default:          scw.ZoneFrPar1,
```
Comment (remyleone) [inline scaleway/resource_lb_ip.go:42]: It should be the zone of the provider.

Comment (Monitob) [reply scaleway/resource_lb_ip.go:42]: Yes, well seen. we should set zoneSchema back.

**Review by remyleone — APPROVED** [review]
